### DISCUSSION TITLE
Thank you for creating and maintaining this library

### DISCUSSION THREAD

**Discussion by saxenanurag** [hozn/stravalib#511] [General]
Hello Maintainers. Just want to say thank you for creating and maintaining this library. I used this library to write an app last year to pull Strava activity data for a bike club so that we could run different competitions. Basically I setup a flask app so that users could sign in with Strava. Once they signed in, I used the stravalib package to pull their activity data once a day using cron. We stored the data in a database and then were able to give out prizes for activities in a time period. Your data models are super helpful. I met Leah at PyCon last week and mentioned this to her as well. Thank you again.

**Comment by lwasser** [comment]
@saxenanurag so good to see you there!! AND thank you for posting this. it's such a funny thing how maintainers get so many reports about bugs and issues and things that aren't working and so little of this type of thing! we need more of this! 

 it was wonderful to meet you at pycon us this year!! i hope the pycon ohio meeting is great and perhaps i'll see you again at pycon us next year!!

**Reply by lwasser** [reply]
oh also @hozn is the original author of this library and worked on it i think for 10 years or so!! pretty impressive!!
